pages/vi/vi-github-and-repositories.md

```diff
@@ -26,44 +26,51 @@ This is just a summary of the steps that you will need to perform. Please, keep
+Now, we will be using GitHub repositories on a command line, which means that there is a separate step to get your GitHub repository onto your OS. To be clear, you will be using both the command line and the online GitHub user interface, meaning you need to constantly check to make sure that your version is not behind to avoid merge conflicts. Therefore, open a command line and open your username.github.io repository on the GitHub user interface. You then need to copy the link provided in the repository (see the picture below).
 
 ![GitHub Clone URL](images/vi-github-clone-url.png)
 
 Then, turn to your command prompt and type your repository URL in the form of `git clone https://github.com/EmilyLarkin/EmilyLarkin.github.io.git` into the command line. Be sure to use the correct URL to clone your repository (you will obviously type your own username).
 
 ##### Clone with HTTPS or Use SSH?
 
-Both HTTPS and SSH URLs identify the same remote repositories but use different protocols to access the codebase. Besides HTTPS, which we talked about above, you can also use SSH to do the same thing. You can explore the differences using [connecting-to-github-with-ssh](https://help.github.com/articles/connecting-to-github-with-ssh/).
+Both HTTPS and SSH URLs identify the same remote repositories but use different protocols to access the codebase. Besides HTTPS, which we talked about above, you can also use SSH to do the same thing. You can explore the differences in thisarticle: [connecting-to-github-with-ssh](https://help.github.com/articles/connecting-to-github-with-ssh/).
 
 ### Explanation About Repositories and Syncing Process
 
 The previous step created a clone of your repository on your OS.
-Now, there are three different Github repository levels: [open-learning-exchange.github.io](https://github.com/open-learning-exchange/open-learning-exchange.github.io), your username.github.io on GitHub, and your username.github.io on your OS. These three levels need to be constantly synced and up to date with one another as we will all be contributing to the upstream repository (open-learning-exchange.github.io). It's important to try and keep these separate and avoid mixing changes between them, as you will be unable to fork and git push/pull if they are very different versions.
+
+Now, there are three different GitHub repository levels:  
+
+  * [open-learning-exchange.github.io](https://github.com/open-learning-exchange/open-learning-exchange.github.io) (*upstream*)  
+  * username.github.io on GitHub  
+  * username.github.io on your OS  
+
+These three levels need to be constantly synced and up-to-date with one another as we will all be contributing to the upstream repository (open-learning-exchange.github.io). It's important to try and keep these separate and avoid mixing changes between them, as you will be unable to fork and git push/pull if they are very different versions.
```

Comment:
I like separating the three level into bullet points. Maybe it would be better to put `The previous step created a clone of your repository on your OS. Now, there are three different GitHub repository levels: ` on the same line. 

Also for bullet points and generally maintaining the md wiki, it is cleaner omit the empty lines. It will have the same format on the wiki. See this comment by @xyb994 https://github.com/open-learning-exchange/open-learning-exchange.github.io/pull/1264

Reply:
In some markdown implementations "-" renders bullets. Apparently this is not the case with GitHub?

Reply:
Not sure I understand the addition of  *Origin* Are you trying to distinguish between Remote and Local?

Reply:
That was my intention; is it unnecessary?